README.md

```diff
@@ -243,6 +243,7 @@ alertmanager:
   # expiresafter: "" if set to a non-zero value, alert expires after that time in seconds (default: 0)
   # extralabels: "" # comma separated list of labels composed of a ':' separated name and value that is added to the Alerts. Example: my_label_1:my_value_1, my_label_1:my_value_2
   # extraannotations: "" # comma separated list of annotations composed of a ':' separated name and value that is added to the Alerts. Example: my_annotation_1:my_value_1, my_annotation_1:my_value_2
+  # severitymap: "" # comma separated list of tuple composed of a ':' separated priority and severity that is used to insert a severity label based on the priority level of falco event. Example: debug:value_1,critical:value2. Default mapping: emergency|alert|critical|error|warning|notice|informational|debug:critical|critical|critical|warning|warning|informational|informational|informational
```

Comment:
I don't understand the example, it doesn't follow the comment

Reply:
in the README it's `severytmap` here and `CUSTOM_SEVERITYMAP` in the en vars,  in the config_example it's `customseveritymap`, and in the code base it's  `SeverityMap`.

Reply:
Sorry, I messed up my commit :/
the default mapping line is representing the (now called) defaultSeverityMap variable.
I switched to CustomSeverityMap to make it more clear that default mapping is always used, and that the custom mapping doesn't not override the whole default map but the common fields.